README.md

```diff
@@ -97,6 +97,7 @@ form](https://coder.com/contact) or send an email to support@coder.com.
 | postgres.sslMode | string | Determines how the connection is made to the database. The acceptable values are: `disable`, `allow`, `prefer`, `require`, `verify-ca`, and `verify-full`. | `"require"` |
 | postgres.useDefault | bool | Deploys an internal Postgres instance alongside the. platform. We don't recommend running the internal Postgres instance in production. If true, all other values are ignored. | `true` |
 | postgres.user | string | the user of the external postgres instance. | `""` |
+| serviceAccount.annotations | object | Adds annotations to the service account used for all Coder deployed services. | `{}` |
```

Comment:
just wondering - should we also allow labels to be customizable?

Reply:
this seems reasonable